### PR TITLE
Individual fields whitelisting option added.

### PR DESCRIPTION
**Pull request by tractorcow**
Replaces #300

### CODE REVIEW

File: src/Extension/FluentExtension.php [hunk 1]
```diff
@@ -23,18 +23,28 @@
 /**
  * Basic fluent extension
  *
+ * When determining whether a field is localised, the following config options are checked in order:
+ * - translate (uninherited, for each class in the chain)
+ * - field_exclude
+ * - field_include
+ * - type_exclude
+ * - type_include
```
Comment (mfendeksilverstripe) [inline src/Extension/FluentExtension.php]: I believe this should be called `data_exclude` and `data_include` to be consistent with the rest of the code.

**Comment by mfendeksilverstripe** [comment]
Additionally, the tests are failing with PHP 5.6.

**Comment by robbieaverill** [comment]
PHPCS violation fixed

File: src/Extension/FluentExtension.php (hunk 1 above)
Comment (robbieaverill) [reply src/Extension/FluentExtension.php]: Yep - seems like a typo. I've pushed a commit that fixes that.

**Comment by tractorcow** [comment]
Ok fixed.

**Pull request merged**